### PR TITLE
Documentation/access controls ac 06

### PR DESCRIPTION
**Pull request by carltonsmith**
### This pull request changes...

Draft Implementation statement for AC-06.

### This pull request is ready to merge when...

- [x] Meets acceptance criteria for the issue 
- [x] Code is reviewed by someone other than the original author
- [x] Code is tested 
- [x] Experience is approved by UX  
- [x] Meets [a11y checklist](https://github.com/HHS/TANF-app/blob/main/docs/a11y/how-18f-will-test-a11y.md)
- [x] Meets [Raft's Manual QASP Checklist](https://teams.microsoft.com/l/file/B5285301-13CA-46FD-A723-AFFD578C224A?tenantId=d58addea-5053-4a80-8499-ba4d944910df&fileType=docx&objectUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA%2FShared%20Documents%2FGeneral%2FManual%20Testing%20Practices_Issue173.docx&baseUrl=https%3A%2F%2Fhhsgov.sharepoint.com%2Fsites%2FTANFDataPortalOFA&serviceName=teams&threadId=19:f769bbcb029f4f02b55ae7fad90e310d@thread.skype&groupId=41f194a6-c1d3-4680-933e-c8ee7d17e287):lock: (only applicable for PR to main HHS)
- [x] Documentation is updated 
  - OpenAPI  
  - Readme
  - Entity Relationship Diagram (ERD)

### CODE REVIEW

**Comment by hursey013** [comment]
@rnafarrete It looks like AC-6 has a number of enhancements (AC-6(1), AC-6(2), AC-6(5), AC-6(9), AC-6(10)) - does this PR specifically address the base AC-6 control?

**Comment by rnafarrete** [comment]
> @rnafarrete It looks like AC-6 has a number of enhancements (AC-6(1), AC-6(2), AC-6(5), AC-6(9), AC-6(10)) - does this PR specifically address the base AC-6 control?

Yes, this is specifically addressing the base AC-6 control.  This is based on the key controls in the sca preassessment checklist.

File: docs/controls/access_control/ac-06.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+AC-6 (2) Additional FedRAMP Requirements and Guidance: Examples of security functions include but are not limited to: establishing system accounts, configuring access authorizations (i.e., permissions, privileges), setting events to be audited, and setting intrusion detection parameters, system programming, system and security administration, other privileged functions.
+
+### TDP Implementation
+
+In order to access the TDP application, users must login through Login.gov, using password and Multi-Factor Authentication (MFA).  Only authorized users can access TDP.
+
+AC-6 (2)  
+Once users are logged in the application, users can only perform tasks associated to their user type as described in [AC-5, Part (b)](docs/controls/access_control/ac-05.md).  Privileged functions can only be performed by OFA Admins (e.g. approves new users, updates profile information, and inactivates/reactivates users) and Data Preppers (e.g. upload and download data for STTs).
```
Comment (hursey013) [inline docs/controls/access_control/ac-06.md:13]: Is uploading and downloading data for STTs considered a privileged function?

**Review by hursey013 — CHANGES_REQUESTED** [review]
This is a great start - I think it might be worth reiterating that new user accounts are established with no privileges upon creation and that OFA Admins are the only ones that can grant permissions to those users.

File: docs/controls/access_control/ac-06.md (hunk 1 above)
Comment (rnafarrete) [reply docs/controls/access_control/ac-06.md]: Nope - good catch.  I removed the data prepper (STT) list of functions.

**Comment by rnafarrete** [comment]
Updated ac-6:
- removed mention of data preppers (stt) functions because they do not have priv. functions
- added "New user accounts are established with no privileges upon creation and only OFA Admins can grant permissions to those users."

**Comment by kniz-raft** [comment]
@hursey013 @lfrohlich is this ready for Tijan and Penyin?

File: docs/controls/access_control/ac-06.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+AC-6 (2) Additional FedRAMP Requirements and Guidance: Examples of security functions include but are not limited to: establishing system accounts, configuring access authorizations (i.e., permissions, privileges), setting events to be audited, and setting intrusion detection parameters, system programming, system and security administration, other privileged functions.
+
+### TDP Implementation
+
+In order to access the TDP application, users must login through Login.gov, using password and Multi-Factor Authentication (MFA).  Only authorized users can access TDP.  
+
+AC-6 (2)  
+Once users are logged in the application, users can only perform tasks associated to their user type as described in [AC-5, Part (b)](docs/controls/access_control/ac-05.md).  Privileged functions can only be performed by OFA Admins (e.g. approves new users, updates profile information, and inactivates/reactivates users).  New user accounts are established with no privileges upon creation and only OFA Admins can grant permissions to those users.
```
Comment (hursey013) [inline docs/controls/access_control/ac-06.md:13]:
```suggestion
Once users are logged in the application, users can only perform tasks associated to their user type as described in [AC-5, Part (b)](docs/controls/access_control/ac-05.md).  Privileged functions can only be performed by OFA Admins (e.g. approves new users, updates profile information, and inactivates/reactivates users).  New user accounts are established with no privileges upon creation and only System Admins can grant permissions to those users.
```
```suggestion
Once users are logged in the application, users can only perform tasks associated to their user type as described in [AC-5, Part (b)](docs/controls/access_control/ac-05.md).  Privileged functions can only be performed by OFA Admins (e.g. approves new users, updates profile information, and inactivates/reactivates users).  New user accounts are established with no privileges upon creation and only OFA Admins can grant permissions to those users.
```

**Review by hursey013 — APPROVED** [review]

**Comment by hursey013** [comment]
@tdrammeh1 @PenyinAsare this ready for your review

**Comment by hursey013** [comment]
Making a note of @tdrammeh1's comment that was accidentally left on another PR: https://github.com/HHS/TANF-app/pull/181#issuecomment-780787663 - @lfrohlich this one looks good to go!

**Pull request merged**